Source/Scene/Primitive.js

```diff
@@ -1077,31 +1103,32 @@ define([
     function createShaderProgram(primitive, frameState, appearance) {
         var context = frameState.context;
```

Comment:
I should note that primitives that supply their own `_createShaderProgramFunction` do not support shadows right now. I believe this is just `GroundPrimitive`.